### ISSUE TITLE
What is the real mAP of this implementation ?

### ISSUE THREAD

**Issue by PiggyGenius** [eriklindernoren/PyTorch-YOLOv3#51]
I think the mAP you mention on the readme of the project is the one you computed with your old evaluation which wasn't a mAP computation. I know there has been a new implementation recently and I used it to compute a mAP on my own version of YOLOv3. The value I get for a 0.5 threshold (VOC mAP) is 0.65 with your mAP implementation. I also wrote a mAP evaluation module and mine gives me a 0.41 VOC mAP. All these evaluations are done on the official weights and on the COCO dataset with 416x416 inputs.

The official score is 0.55, my evaluation might be wrong or it might be that my darknet implementation is missing things / have small mistakes. The fact that your evaluation gets such a high score tells me that it is almost certainly wrong though. The last option is that the author made a mistake himself which I would consider unlikely. I will review the code soon and complete the issue if I find mistakes but I thought I would mention it now since your mAP numbers are already in the readme.

Another completely different possibility is that your evaluation module is correct and the high score is explained by the fact that I use the validation set of 2014 and that the author trained the official weights on them. Which might be the most likely case because I just noticed that in the script he uses to download the coco dataset he creates a validation subset of 5k images.

**Comment by lolongcovas** [comment]
I executed 
```
python3.6 test.py --batch_size 10 --n_cpu 8
```
and gave me an mAP of 0.40974596199132984

with
```
python3.6 test.py --batch_size 10 --n_cpu 8
```
gave me an mAP of 0.4856844428591997

how could you achieve 0.58 in mAP?
